spec/acceptance/nginx_server_spec.rb

```diff
@@ -11,7 +11,10 @@ class { 'nginx': }
         ensure   => present,
         www_root => '/var/www/www.puppetlabs.com',
       }
-      host { 'www.puppetlabs.com': ip => '127.0.0.1', }
+      file_line {'mock hosts':
+        path => '/etc/hosts',
+        line => '127.0.0.1 www.puppetlabs.com'
+      }
```

Comment:
Looks like `host_core` is not in fixtures.yml.

While this is indeed an extra dependency for CI, I like the idea of acceptance tests that follow best practices by using relevant modules: if someone wants to run nginx on windows (oh my), they will probably have to revert this.  But this is definitively not a blocker.

Reply:
I actually did implement that with `host_core` and the removed it again because I could not find any way to add a fixture but **only** for acceptance tests.
I will open another issue for it